script/deploy/RelayDeploy.sol

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {Script} from "forge-std/Script.sol";
+
+import {CreateXWrapper} from "@symbioticfi/core/script/utils/CreateXWrapper.sol";
+import {SymbioticCoreConstants} from "@symbioticfi/core/test/integration/SymbioticCoreConstants.sol";
+
+import {TransparentUpgradeableProxy} from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+
+import {IOpNetVaultAutoDeploy} from "../../src/interfaces/modules/voting-power/extensions/IOpNetVaultAutoDeploy.sol";
+import {VotingPowerProvider} from "../../src/modules/voting-power/VotingPowerProvider.sol";
+
+abstract contract RelayDeploy is Script, CreateXWrapper {
+    // CREATE3 salts
+    bytes32 public VOTING_POWERS_SALT = keccak256("VotingPowers");
+    bytes32 public KEY_REGISTRY_SALT = keccak256("KeyRegistry");
+    bytes32 public SETTLEMENT_SALT = keccak256("Settlement");
+    bytes32 public DRIVER_SALT = keccak256("Driver");
+
+    function _settlementParams() internal virtual returns (address implementation, bytes memory initData);
+    function _votingPowerParams() internal virtual returns (address implementation, bytes memory initData);
+    function _keyRegistryParams() internal virtual returns (address implementation, bytes memory initData);
+    function _driverParams() internal virtual returns (address implementation, bytes memory initData);
+
+    function run() public virtual {
+        deploySettlement();
+        deployVotingPower();
+        deployKeyRegistry();
+        deployDriver();
+    }
+
+    function deployVotingPower() public virtual returns (address) {
+        vm.startBroadcast();
+        (address implementation, bytes memory initData) = _votingPowerParams();
+        address newContract = _deployContract(VOTING_POWERS_SALT, implementation, initData);
+        vm.stopBroadcast();
+
+        // Validate deployment
+        SymbioticCoreConstants.Core memory symbioticCore = SymbioticCoreConstants.core();
+        require(
+            VotingPowerProvider(newContract).OPERATOR_REGISTRY() == address(symbioticCore.operatorRegistry),
```

Comment:
Anvil doesn't have constants for core, so need to add "coreSupported" method to  SymbioticCoreConstants